README.md

```diff
@@ -34,12 +34,16 @@ If you wish to use the pthreads implementation on Windows, in preference to the
 native win32 one, you need to define `C11THREADS_PTHREAD_WIN32`.
 
 ### With Win32 threads
-To use C11 threads over the Windows threads API, beyond adding `c11threads.h` to
-your project, you also need to compile `c11threads_win32.c` as part of your
-build. Additionally, if you're in a situation where letting c11threads keep
-resources for the duration of the process lifetime is not desirable, you can
-call `c11threads_destroy_win32()` to free them manually at any point, when
-you're done with it.
+
+Simply add the CMake project to your own project or build a shared library from
```

Comment:
I don't like the cmake-specific wording. I prefer instructing people to compile `c11threads_win32.c`, and leave them to do it with whatever build system they choose to.

If there is some valueable advice for cmake users, it should be less prominent, in the form of "if you use cmake, you can do X Y and Z", but if it's not some arcane knowledge we're talking about, and it's plain enough what needs to be done, I'd rather leave it at the build-system agnostic instructions.